.github/pull_request_template.md

```diff
@@ -69,6 +69,11 @@ Common, optional tasks are included here in case you forgot something important.
 - Are there any new queries in your change set that might require new indexes?
 - Do any new queries require time-boxing to avoid table-scans when the data grows?
 
+
+##### Data & Analytics Dependencies
+- For customer-facing applications, are there any GA tagging changes that are needed or created by this change? If so, have those been discussed with D&A?
+- Has Data Engineering been notified of any schema changes?
```

Comment:
This should mainly be for modifications of existing schema attributes. Additions shouldn't matter. I don't know the details of what was discussed previously around this though.

Reply:
"Has Data Engineering been notified of any schema changes?" Should cover all the basis  here.

Reply:
> Has Data Engineering been notified of any schema changes
Not sure this is the best wording or requirement. We do not do that unless it results in an impact on Datafuse configuration. Even if it does, we tend to prepare a PR to change that and run that true Data team.